docs/iris/src/whatsnew/latest.rst

```diff
@@ -153,6 +177,17 @@ This document explains the changes made to Iris for this release
   :func:`iris.analysis.trajectory.interpolate` function.  This prevents
   duplicate coordinate errors in certain circumstances. (:pull:`3718`)
 
+* `@bjlittle`_ aligned the :func:`iris.analysis.maths.apply_ufunc` with the
+  rest of the :mod:`iris.analysis.maths` API by changing its keyword argument
+  from ``other_cube`` to ``other``. (:pull:`3785`)
+
+* `@bjlittle`_ changed the :meth:`iris.analysis.maths.IFunc.__call__` to ignore
+  any surplus ``other`` keyword argument for a ``data_func`` that requires
+  **only one** argument. This aligns the behaviour of
+  :meth:`iris.analysis.maths.IFunc.__call__` with
```

Comment:
```suggestion
  :meth:`~iris.analysis.maths.IFunc.__call__` with
```

Reply:
@trexfeathers I intentionally didn't put the `~` in otherwise it collapsed down to `__call__` which I find pretty useless, and you're forcing the user to follow the link (or hover over it) to see what function it's talking about.

What do you think?

Reply:
Ah! The lesser of two evils then. I agree 😊

Reply:
Yeah, I flipped and flopped with that one... but decided that being explicit may be more helpful since it was a dunder method and not explicitly part of the public API :+1: